### DISCUSSION TITLE
Osu lazer! don't play "see you next time" message when leaving

### DISCUSSION THREAD

**Discussion by Flex146** [ppy/osu#19573] [Q&A] [answered]
When i leave osu lazer it doesn't say "see you next time", it just leaves the game, i using the back button on my phone but it just exit the app without the mesaage
I'm on android 10, osu lazer version: 2022.731.1

**Comment by peppy** [comment, marked as answer]
This is correct behaviour. Mobile apps don't "exit"

**Reply by nagi-desuuu** [reply]
For more info, see https://github.com/ppy/osu/discussions/16546

**Reply by Flex146** [reply]
i see... thanks anyways
